csharp/test/Apache.Arrow.Tests/TestMemoryAllocator.cs

```diff
@@ -13,17 +13,54 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+using System;
 using Apache.Arrow.Memory;
 using System.Buffers;
+using System.Diagnostics;
+using System.Threading;
+using Xunit;
 
 namespace Apache.Arrow.Tests
 {
     public class TestMemoryAllocator : MemoryAllocator
     {
+        private int _rented = 0;
```

Comment:
Could the formatting in this class please be tidied up by adding blank lines around methods.